vendor/bat-native-ads/src/bat/ads/internal/experiments/page_probabilities/page_probabilities_experiment.cc

```diff
@@ -0,0 +1,67 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "bat/ads/internal/experiments/page_probabilities/page_probabilities_experiment.h"
```

Comment:
for future reference when defining classes the file name must match the class name so this would either be page_probabilities.cc or the class name would be PageProbabilitiesExperiment (although this is actually a study, not an experiment)

Reply:
resolved with refactor